omnia.yml

```diff
@@ -50,6 +50,9 @@
 - name: Import playbook to install ROCm on nodes
   ansible.builtin.import_playbook: utils/rocm_installation.yml
 
+- name: Verify Gaudi nodes
+  ansible.builtin.import_playbook: utils/verify_intel_gaudi/verify_intel_gaudi_installation.yml
```

Comment:
this playbook should be invoked based on variable in omnia_config.yml

Reply:
Go back to the way it was before (see first comment in this PR) ? or do you have another suggestion

Reply:
we can revert the changes...it should invoke based on variable verify_intel_gaudi_with_sanity_test_only

Reply:
done